polaris/polaris/management/commands/process_pending_deposits.py

```diff
@@ -729,14 +832,202 @@ async def save_as_pending_signatures(cls, transaction, server):
             transaction.envelope_xdr = envelope.to_xdr()
             transaction.pending_signatures = True
             transaction.status = Transaction.STATUS.pending_anchor
-        transaction.pending_execution_attempt = False
         await sync_to_async(transaction.save)()
         await maybe_make_callback_async(transaction)
 
     @classmethod
     def handle_error(cls, transaction, message):
         transaction.status_message = message
         transaction.status = Transaction.STATUS.error
-        transaction.pending_execution_attempt = False
         transaction.save()
         logger.error(message)
+
+    @classmethod
+    def handle_transaction_submission_exception(cls, transaction, exception, message):
```

Comment:
Can we rename this to `update_submission_status()` or something similar? Its just a bit more explicit.